### ISSUE TITLE
[BUG] Dokumentennamen-Erweiterung

### ISSUE THREAD

**Issue by Dickmaulruessler** [jlawyerorg/j-lawyer-org#1122]
Nicht wirklich ein "Bug", aber ...

...ganz am Anfang bei den ersten Schritten mit j-lawyer habe ich in einer Akte testweise mehrere Dokumente erstellt und war dann davon "genervt", dass man ab dem zweiten Dokument (wenn das alles am selben Tag erfolgt) den Dokumentennamen ergänzen muss. Ich wäre davon ausgegangen, dass automatisch die Dokumentennummer hochgezählt wird. Lässt sich das mit wenig Arbeit realisieren? 
In der Praxis ist das aber glaube ich kein Problem, da man an denselben Beteiligten nicht am selben Tag mehrere Schriftsätze schreibt. 

Etwas (kleines) anderes: Wäre es nicht logischer, die Zählung begänne mit "1" statt mit "0"?

Jens, bitte setze das gleich wieder auf "closed", falls das mehr als 5 Minuten Arbeit bedeutet.

**Comment by stm9x9** [comment]
> ...ganz am Anfang bei den ersten Schritten mit j-lawyer habe ich in einer Akte testweise mehrere Dokumente erstellt und war dann davon "genervt", dass man ab dem zweiten Dokument (wenn das alles am selben Tag erfolgt) den Dokumentennamen ergänzen muss. 

Muß man das? Wird nicht vorne noch Datum und Uhrzeit gesetzt?

**Comment by Dickmaulruessler** [comment]
nein, Uhrzeit nicht. Das wäre natürlich auch eine Alternative.

**Comment by j-dimension** [comment]
> Etwas (kleines) anderes: Wäre es nicht logischer, die Zählung begänne mit "1" statt mit "0"?

@Dickmaulruessler kannst Du sagen was für eine Zählung Du meinst?



> Jens, bitte setze das gleich wieder auf "closed", falls das mehr als 5 Minuten Arbeit bedeutet.

KEIN Ticket gibt es für 5min, dann müsste ich auf jegliche Tests verzichten :-)

**Comment by Dickmaulruessler** [comment]
> KEIN Ticket gibt es für 5min, dann müsste ich auf jegliche Tests verzichten :-)

Ja, das war mir auch bewusst. Ich wollte nur deutlich zum Ausdruck bringen, dass das ein Mini-Luxux-Problem darstellt (ganz am Anfang bei den ersten Schritte aber wirklich genervt hatte)

**Comment by Dickmaulruessler** [comment]
> 
> @Dickmaulruessler kannst Du sagen was für eine Zählung Du meinst?
> 
![dsafdsaf](https://user-images.githubusercontent.com/46451749/107855109-d4046a00-6e20-11eb-9f6a-f52d7e21beec.jpg)

**Comment by j-dimension** [comment]
> > @Dickmaulruessler kannst Du sagen was für eine Zählung Du meinst?
> 
> ![dsafdsaf](https://user-images.githubusercontent.com/46451749/107855109-d4046a00-6e20-11eb-9f6a-f52d7e21beec.jpg)

Ähm... die kommt nicht aus der Anwendung :-)   
2021-02-13_    ist der automatische Präfix, die "0" muss bereits Teil des Vorlagenamens sein.

**Comment by Dickmaulruessler** [comment]
ahhhhhhhhhhhhhhhhhh! Stimmt. (ich habe die "0" da, damit die Vorlage bei mir ganz oben steht)

Dann wäre die Frage, ob man den Präfix um eine hochzählende Nr. ergänzen könnte oder die Sache schließen sollte.

**Comment by j-dimension** [comment]
Kann man machen, so einen Zähler.

**Comment by Dickmaulruessler** [comment]
Bei der Notizfunktion wäre das ebenso (ich weiß nicht, ob eine Änderung bei den normalen Vorlagen sich auch hier auswirken würde). 
Bei den Notizen könnte das praktisch relevanter sein, da man davon ja evtl mal mehrere pro Tag erstellt, da immer um Ergänzung des Dateinamens gebeten wird und man gerade hier, z.B. während eines Telefonats, sich in besonderer Eile befindet und nur mit einer Hand dabei ist.

**Comment by Dickmaulruessler** [comment]
In den Notizen taucht im Dateinamen jetzt (Vers. 3.0) neuerdings die Uhrzeit auf. 
Bei anderen Dateien weiterhin ohne Uhrzeit. 
Kann man den Dateinamen speziell für die Notizen konfigurieren? 
Unter "Einstellungen / Dokumente / Dateinamen" habe ich das nicht gefunden. (werden die Notizen möglicherweise wie E-Mail behandelt?).

**Comment by j-dimension** [comment]
> Kann man den Dateinamen speziell für die Notizen konfigurieren?

Nein, das ist aktuell nicht möglich.

**Comment by iradraconis** [comment]
Es gab mal ein Ticket, wonach es explizit gewünscht war, die Uhrzeit in den Dateinamen aufzunehmen, für den Fall, dass mehrere Notizen an einem Tag erstellt werden. Ich hatte das dann so umgesetzt.
